web/src/app/router.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const konnektApp = angular.module('konnektApp', ['ngRoute']);
```

Comment:
this creates a new `konnektApp` module instead of using the existing one, remove the second parameter from the module method